app/src/components/template/listItems.js

```diff
@@ -7,38 +7,71 @@ import Message from '@material-ui/icons/Message';
 import BarChartIcon from '@material-ui/icons/BarChart';
 import LayersIcon from '@material-ui/icons/Layers';
 import AssignmentIcon from '@material-ui/icons/Assignment';
+import { NavLink } from 'react-router-dom';
+import '../../styles/index.css';
 
 export const mainListItems = (
     <div>
-        <ListItem button>
-            <ListItemIcon>
-                <DashboardIcon />
-            </ListItemIcon>
-            <ListItemText primary={'Home'} />
-        </ListItem>
-        <ListItem button>
-            <ListItemIcon>
-                <AssignmentIcon />
-            </ListItemIcon>
-            <ListItemText primary={'OrderBook Snapshot'} />
-        </ListItem>
-        <ListItem button>
-            <ListItemIcon>
-                <Message />
-            </ListItemIcon>
-            <ListItemText primary={'Message List'} />
-        </ListItem>
-        <ListItem button>
-            <ListItemIcon>
-                <BarChartIcon />
-            </ListItemIcon>
-            <ListItemText primary={'Squarepoint Orders'} />
-        </ListItem>
-        <ListItem button>
-            <ListItemIcon>
-                <LayersIcon />
-            </ListItemIcon>
-            <ListItemText primary={'TBD'} />
-        </ListItem>
+        <NavLink
+            exact
+            to={'/'}
```

Comment:
I don't think a sidebar button should bring us to the homepage, maybe move this behavior to the "Graphelier" on the navbar